D1673/P1673.md

```diff
@@ -8756,21 +8756,9 @@ void matrix_rank_1_update(
 `xGER` (for real element types) and
 `xGERU` (for complex element types). <i>-- end note]</i>
 
-[1]{.pnum} *Preconditions:*
-
-  * [1.1]{.pnum} `A.extent(0)` equals `x.extent(0)`, and
+[2]{.pnum} *Mandates:* _`possibly-multiplyable`_`<InOutMat, y, x>()` is `true`.
```

Comment:
```suggestion
[2]{.pnum} *Mandates:* _`possibly-multipliable`_`<InOutMat, y, x>()` is `true`.
```

Reply:
It's $A = A x y^T$

Reply:
InMatrix is less constraint than InOutMatrix so it does match. (for the record)